doc/cephfs/fs-volumes.rst

```diff
@@ -1046,6 +1046,120 @@ following command.
 
 .. _subvol-pinning:
 
+Controlling Subvolume Snapshot Visibility
+-----------------------------------------
+
+.. note:: This functionality is currently supported only for FUSE clients.
+          Kernel client support is planned; progress can be tracked at
+          https://tracker.ceph.com/issues/72589.
+
+Snapshots of a subvolume can be hidden from compatible clients by
+performing two actions:
+    1) Disable the subvolume's ``snapshot_visibility`` flag (default is ``true``).
```

Comment:
Suggest

Set the subvolume's ``snapshot_visiblity`` flag to ``false`` (default is ``true``).